backend/src/main/java/planit/people/preparation/GoogleConnector/GoogleConnector.java

```diff
@@ -38,7 +36,7 @@ public class GoogleConnector {
      * Global instance of the JSON factory.
      */
     private static final JsonFactory JSON_FACTORY = GsonFactory.getDefaultInstance();
-    private static final String CREDENTIALS_FILE_PATH = "backend/src/main/java/planit/people/preparation/Google_Connector/client_secret.json";
+    private static final String CREDENTIALS_FILE_PATH = "backend/src/main/resources/client_secret.json";
```

Comment:
Can we include changes from backend/src/main/java/planit/people/preparation/Google_Connector/Google_Connector.java in this file?

Reply:
<img width="1577" alt="image" src="https://user-images.githubusercontent.com/6399265/195854517-6e3f4186-c125-4f25-91cc-ac4afb744ce3.png">